core/chains/evm/txm/docs/TRANSACTION_MANAGER_V2.md

```diff
@@ -0,0 +1,13 @@
+
+# Transaction Manager V2
+
+## Configs
+- `EIP1559`: enables EIP-1559 mode. This means the transaction manager will create and broadcast Dynamic attempts. Set this to false to broadcast Legacy transactions.
+- `BlockTime`: controls the interval of the backfill loop. This dictates how frequently the transaction manager will check for confirmed transactions, rebroadcast stuck ones, and fill any nonce gaps. Transactions are getting confirmed only during new blocks so it's best if you set this to a value close to the block time. At least one RPC call is made during each BlockTime interval so the absolute minimum should be 2s. A small jitter is applied so the timeout won't be exactly the same each time.
```

Comment:
If the absolute minimum should be 2s, is it maybe worth adding a config validation so we can fail fast? If we still want to leave it up to the user, maybe we should phrase this as a recommended minimum?